### ISSUE TITLE
Ability to display text value even if the translation key doesn't exist

### ISSUE THREAD

**Issue by JuradoBruno** [lhz516/react-simple-i18n#15]
First, thanks for this great and very light library.

Issue description:
When I create an App, I like to not worry about translation keys until the 1st production deployment.
That said, when it is time to handle translations, I need a way to quickly find every text that need translations so all along my coding I'm still using the translation module but with the english key result, eg:
`<h1 className="text-2xl">{t("Users")}</h1>`
This way, later I can't search and find every `t("` instances and create translation keys accordingly, for this example, `t("Users")` will become something like `t("layout.organizations.users.title")`.

I use this process so I can have a global view of my project and create the translation key structures accordingly. Also It allows to stay focused on the same task longer without switching context, hence more efficiency.

The issue with this module is that it returns an empty string when the key doesn't exists in the translation object, forcing us to create the key each time we add text to our UI.

I'm giving you the solution I used so you can consider adding something similar, or not. 

In the file **create-i18n.ts**, in the method **getNestedValue()**:

```
export const getNestedValue = (data: any, keyArr: Array<string>, currentIndex = 0): string => {
  if (currentIndex === keyArr.length) {
    if (typeof data === "string") {
      return data;
    } else {
      return getMissingTranslation(keyArr[currentIndex - 1]);   <------------------------------
    }
  }

  function getMissingTranslation(missingKey: string) {
    return `${missingKey}`;
  }

  return getNestedValue(data[keyArr[currentIndex]], keyArr, currentIndex + 1);
};
```

Another benefit of this feature is the user could modify the content of **getMissingTranslation()** by adding a "*" or "@" to the return object, so he can easily spot all the missing translations in the UI.

Either way, I feel like showing the text used in the brackets instead of an empty string when the translation is missing is the expected behavior (at least it was mine, based on the use of other trad libraries).

Cheers

**Comment by lhz516** [comment]
Sounds good. I saw some other libs use the i18n key itself as the default text. Eg. If `t('submit_button')` doesn't exist, the text in the DOM will be `submit_button`.

I believe we can follow the above behavior and also add an option for custom default text. The option is a function which takes the i18n `key` as an argument. Something looks like:
```jsx
<I18nProvider i18n={createI18n(langData, {
  lang: 'enUS'
  defaultText: (key) => {
    return `MISSING_${key}`;
  }
})}>
```
The `defaultText` option is optional. It defaults to `(key) => key` which displays the key string directly.
I can add this feature hopefully by end of the week.

**Comment by JuradoBruno** [comment]
Yes that would be amazing, thanks mate 💪

**Comment by lhz516** [comment]
@JuradoBruno Published `1.4.0-beta.0`

I haven't tested it in a real project. Feel free to try it out.
